app/webpacker/stylesheets/components/_external-content.scss

```diff
@@ -13,6 +13,7 @@
   h2,
   h3 {
     @extend .heading-generic;
+    font-family: 'Objectivity', sans-serif;
```

Comment:
as discussed, I think this would be better set on `.govuk-heading-m` since I don't think we want this font on every h3, one to double check with @maddyjoe if you aren't sure, also might be worth doing a tiny design check to verify it's only impacted the expected headings?

Reply:
I don't think the standard h3 should be using this font too?

Reply:
resolved too thank you 😊